python/tests/scripts/f1_imports.metta

```diff
@@ -1,13 +1,19 @@
 ; NOTE: this behavior can change in the future
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-; Even at the very beginning of the main script `(get-atoms &self)`
-; returns one atom, which wraps the space of stdlib.
+; At the very beginning of the main script, the space
+; contains an atom wrapping the space of stdlib.
 ; The type of this atom is the same as of `&self`
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; QUESTION: Are space implementations really required to preserve ordering of atoms?
+; QUESTION / TODO: Is there a better way to find this sub-space than `get-atoms`?
+;    Now that we load the init.metta file as a module, the previous comment about the
+;    &self space containing only one atom at the beginning is no longer true
```

Comment:
My idea was to have a library manager (which was partly implemented) interfaced to MeTTa, so one could have an analog of Python `__modules__`, etc. Apparently, we have a huge issue regarding imports / modules, and this question should be considered as a part of this large issue. BTW, I think that we need some way to have systematic design discussions instead of considering different yet related issues separately. Maybe, having tracking issues on github may work, but maybe there is a better technical way to structure these discussions.

Reply:
It's a non-issue now because the test environment no longer loads the init.metta.  I hope we can find a better way to test module loading than relying on specific atom ordering in the space, but for now I have reverted the test changes back to the previous state in https://github.com/trueagi-io/hyperon-experimental/commit/dc15fc359ebb7b7dad9014891da563a6dfda709d